fastplotlib/layouts/_figure.py

```diff
@@ -543,6 +543,10 @@ def show_tooltips(self) -> bool:
         """show/hide tooltips for all graphics"""
         return self._show_tooltips
 
+    @property
+    def animation_funcs(self) -> list[callable]:
```

Comment:
might be useful to instead return a dict of lists `{"pre": list[callables], "pos": list[...]}`, or a tuple `(pre_list, post_list)` ? 

and docstring